model/src/test/kotlin/AdvisorRecordTest.kt

```diff
@@ -97,7 +98,7 @@ class AdvisorRecordTest : WordSpec({
             record.hasIssues shouldBe false
         }
 
-        "detect a result that has issues" {
+        "return true given the record has issues" {
```

Comment:
Please move this fixup to the first commit in this PR, which introduced this line.